packages/text-field/src/TextFieldPresenter.js

```diff
@@ -159,7 +164,10 @@ export default class TextFieldPresenter extends Component {
         >
           <div
             className={cx("hig__text-field-v1__input-wrapper", {
-              "hig__text-field-v1__input-wrapper--focused": this.state.focused,
+              "hig__text-field-v1__input-wrapper--focused":
+                this.props.focused !== undefined
```

Comment:
This code is duplicated below. We can [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) this up by assigning it to a constant above the return.